ethereum/eth/src/main/java/org/hyperledger/besu/ethereum/eth/manager/PeerReputation.java

```diff
@@ -30,21 +30,32 @@
 import org.slf4j.LoggerFactory;
 
 public class PeerReputation implements Comparable<PeerReputation> {
+  static final long USELESS_RESPONSE_WINDOW_IN_MILLIS =
+      TimeUnit.MILLISECONDS.convert(1, TimeUnit.MINUTES);
+  static final int DEFAULT_MAX_SCORE = 100;
   private static final Logger LOG = LoggerFactory.getLogger(PeerReputation.class);
   private static final int TIMEOUT_THRESHOLD = 3;
   private static final int USELESS_RESPONSE_THRESHOLD = 5;
-  static final long USELESS_RESPONSE_WINDOW_IN_MILLIS =
-      TimeUnit.MILLISECONDS.convert(1, TimeUnit.MINUTES);
 
   private final ConcurrentMap<Integer, AtomicInteger> timeoutCountByRequestType =
       new ConcurrentHashMap<>();
   private final Queue<Long> uselessResponseTimes = new ConcurrentLinkedQueue<>();
 
-  private static final int DEFAULT_SCORE = 100;
   private static final int SMALL_ADJUSTMENT = 1;
   private static final int LARGE_ADJUSTMENT = 10;
 
-  private long score = DEFAULT_SCORE;
+  private int score;
+
+  private final int maxScore;
+
+  public PeerReputation() {
+    this(DEFAULT_MAX_SCORE);
+  }
+
+  public PeerReputation(final int maxScore) {
+    this.maxScore = maxScore;
+    this.score = maxScore;
```

Comment:
can we set the initial score less than the max? It's useful right now to see if a peer has score = default, then they haven't us sent anything yet

Reply:
update to set an initial (default 100) and a max (default 150) score, defaults could be adapted based on metrics and adjustment size